nav2_bringup/launch/system_test.yaml

```diff
@@ -0,0 +1,9 @@
+map_server:
```

Comment:
@crdelsey In this directory I think they should be combined into a single system_test.yaml file. This would be a convenient list of all of the system parameters that one could override/change, if desired. The whole system could be started with this file and individual nodes can also use the same file. No need to duplicate on a per-node basis (separate YAML files per node).